packages/utils/telemetry-utils/src/utils.ts

```diff
@@ -16,6 +16,8 @@ import { ITelemetryGenericEventExt, ITelemetryLoggerExt } from "./telemetryTypes
  * @param logger - The logger to log with
  * @param event - The string or event to log
  * @returns The outcome of the condition
+ *
+ * @public
```

Comment:
Wants to be internal (or even move to its single callsite in the repo). Confirmed no uses in bohemia nor loopsdk.

Reply:
I don't see any uses of this in the repo outside of tests. I went ahead and marked it as `@deprecated`.

Reply:
Update: I realized my search was wrong. I do see 1 actual use. I'll copy the logic over there privately.